### PR TITLE
Add Current Initiatives, add categorization example

### PR DESCRIPTION
**Pull request by ccooper**

### CODE REVIEW

**Review by milescrabill — APPROVED** [review]
LGTM, well explained

File: README.md [hunk 1]
```diff
@@ -19,25 +19,54 @@ Theme -> Initiative -> Milestone -> Epic -> Issue
 Themes are large focus areas than span the Mozilla organization and are pertinent and addressable by the Taskcluster team.
 
 For Taskcluster @ Mozilla, we are currently tracking the following themes:
-* Making Data Actionable
-* Ease of use
+* Make data actionable
 * Promote external adoption
-* Cost Reduction
+* Ease of use
+* Cost reduction
```
Comment (djmitche) [inline README.md]:
```suggestion
* Improve Ease of Use
* Support Cost Reduction
```

**Review by djmitche — APPROVED** [review]
Nice!

**Review by helfi92 — APPROVED** [review]
💫

**Pull request merged**